src/ui_parts/viewport.gd

```diff
@@ -57,7 +57,7 @@ func _unhandled_input(event: InputEvent) -> void:
 		return
 	
 	if event is InputEventMouseMotion and\
-	event.button_mask in [MOUSE_BUTTON_MASK_LEFT, MOUSE_BUTTON_MASK_MIDDLE]:
+	event.button_mask & (MOUSE_BUTTON_MASK_LEFT | MOUSE_BUTTON_MASK_MIDDLE):
```

Comment:
`event.button_mask == MOUSE_BUTTON_MASK_LEFT or event.button_mask == MOUSE_BUTTON_MASK_MIDDLE`

Would prefer to not have bitwise stuff in the code if not necessary.

Reply:
Its all mask values... why not use bitwise operations for it?

Reply:
Oh I see, this won't work the way I suggested.